server/api/users.go

```diff
@@ -58,7 +58,8 @@ func PatchUser(c *gin.Context) {
 		c.AbortWithStatus(http.StatusNotFound)
 		return
 	}
-	user.Active = in.Active
+
+	// TODO: currently nothing is updatable
```

Comment:
in should be used and only the  `in.id = user.id` to make sure id is not changed ...
or if we allow partial changes... make a check if a field in `in` has a value and replace it here in `user` ...

PS: securety: we make sure only allowe user can change protected fields like Admin etc ...

Reply:
Sure. Currently users are updated by the oauth systems, so in 99% there currently is no need to update an user at all.

Reply:
That's currently nothing as isAdmin is set by WOODPECKER_ADMINS and login, email, avatar are received from the forgea

Reply:
Would be nice if it was possible to set admin option from UI and env variable would be used for initial user creation

Reply:
Yeah definitely. I think we could add some code that sets `isAdmin` at server start for the listed users, but via the UI those setting could be changed later on.

Reply:
I have adjusted the code and now the admin flag is set to true when a user logs in to the UI.